ctx_test.go

```diff
@@ -708,6 +709,30 @@ func Benchmark_Ctx_BodyParser_JSON(b *testing.B) {
 	utils.AssertEqual(b, "john", d.Name)
 }
 
+// go test -v -run=^$ -bench=Benchmark_Ctx_BodyParser_ProblemJSON -benchmem -count=4
+func Benchmark_Ctx_BodyParser_ProblemJSON(b *testing.B) {
+	app := New()
+	c := app.AcquireCtx(&fasthttp.RequestCtx{})
+	defer app.ReleaseCtx(c)
+	type Demo struct {
+		Name string `json:"name"`
+	}
+	body := []byte(`{"name":"john"}`)
```

Comment:
pls use a blueprint for the new problem json structure

Reply:
Since I've changed `ProblemJSON` to `CustomJSON` we can't make any assertions of the JSON body, but I've included a check in this file to ensure Content-Headers that end with 'json' get parsed as valid JSON.